BackEndLib/Wchar.h

```diff
@@ -129,7 +129,7 @@ bool isWInteger(const WCHAR* wcz);
 bool isInteger(const char* pcz);
 
 int      _Wtoi(const WCHAR* wsz);
-WCHAR*   _itoW(int value, WCHAR* wcz, int radix);
+WCHAR*   _itoW(int value, WCHAR* wcz, int radix, int bufferLength = INT_MAX);
```

Comment:
The buffer used in this function has a magic number size of 512, so bufferLength should never be allowed at a value larger than 512.